.storybook/.babelrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Can we use `extends: '@dhis2/cli-app-scripts/config/app.babel.config.js'` here instead of duplicating the plugin list?  I'm also not sure we need anything but the `styled-jsx` plugin (the `app` babel config includes syntax support for other things, but doesn't actually transpile them since CRA already does.  If we need to do the transpiling here (we might, if CRA's webpack doesn't merge configs) then we can use `dhis2/cli-app-scripts/config/babel.config.js` instead.

Reply:
We can do this:

```
{
    "extends": "../node_modules/@dhis2/cli-app-scripts/config/app.babel.config.js"
}
```

Using `@dhis2/cli-app-scripts/config/app.babel.config.js` directly makes it look in the wrong place:

```
Error: Cannot find module '/home/varl/dev/dhis2/libs/ui-core/.storybook/@dhis2/cli-app-scripts/config/app.babel.config.js' from '/home/varl/dev/dhis2/libs/ui-core'
```

Reply:
Could do `.babelrc.js` and use `require.resolve`, might be a bit more resilient if we bake this as a storybook wrapper lib (`../node_modules` doesn't work in monorepos)

Reply:
It has to be named `.babelrc` but using it according the JS config format works.

```
info => Loading custom babel config as JS
```